Add new Page for DR (#7425)
chef/automate: components/docs-chef-io/content/automate/DR.md

    ```sh
    systemctl stop chef-automate
    ```

    - Make sure both backup and restore cron are aligned.

    - Run the following command in one of the Automate nodes to get the IDs of all the backup:

    ```sh
    chef-automate backup list
    ```

    - Configure the cron that triggers the `chef-automate backup restore` on one of the Chef Automate nodes.

        - To run the restore command, you need the airgap bundle. Get the Automate HA airgap bundle from the location `/var/tmp/` in Automate instance. For example: `frontend-4.x.y.aib`.

        - In case the airgap bundle is not present at `/var/tmp`, it can be copied from the bastion node to the Automate frontend node

        - Run the command at the Automate node of Automate HA cluster to get the applied config:

        ```bash
        sudo chef-automate config show > current_config.toml
        ```

        - Add the OpenSearch credentials to the applied config. If using Chef Managed OpenSearch, add the config below into `current_config.toml` (without any changes).

        ```bash
        [global.v1.external.opensearch.auth.basic_auth]
            username = "admin"
            password = "admin"
        ```

>         - To restore backup on Chef Automate HA, below is the restore command, which can be triggered from any Chef Automate instance of the Disaster Recovery cluster. Below the restore command is an example command for the file system as a backup option. Sample cron for restoring backup saved in file system looks like:

Reviewer: ```suggestion
        - In the Disaster Recovery cluster, use the following sample command to restore the latest backup from any Chef Automate frontend instance:
```

        ```cmd

>         id=$(sudo chef-automate backup list | tail -1 | awk '{print $1}')

Reviewer: ```suggestion
        id=$(chef-automate backup list | grep completed | tail -1 | awk '{print $1}')
```

>         sudo chef-automate backup restore /mnt/automate_backups/backups/$id/ --patch-config current_config.toml --airgap-bundle /var/tmp/frontend-4.x.y.aib --skip-preflight

Reviewer: ```suggestion
       chef-automate backup restore /mnt/automate_backups/backups/$id --patch-config /path/to/current_config.toml --airgap-bundle /var/tmp/frontend-4.x.y.aib --skip-preflight
```

        ```

        Sample cron for restoring backup saved in object storage (S3) looks like this:

        ```cmd

>         id=$(sudo chef-automate backup list | tail -1 | awk '{print $1}')

Reviewer: ```suggestion
        id=$(chef-automate backup list | grep completed | tail -1 | awk '{print $1}')
```

>         sudo chef-automate backup restore <backup-url-to-object-storage>/automate/$id/ --patch-config current_config.toml --airgap-bundle /var/tmp/frontend-4.x.y.aib --skip-preflight --s3-access-key "Access_Key"  --s3-secret-key "Secret_Key"

Reviewer: ```suggestion
       chef-automate backup restore <backup-url-to-object-storage>/automate/$id --patch-config /path/tocurrent_config.toml --airgap-bundle /var/tmp/frontend-4.x.y.aib --skip-preflight --s3-access-key "Access_Key"  --s3-secret-key "Secret_Key"
```

        ```

### Switch to Disaster Recovery Cluster

Steps to switch to the disaster recovery cluster are as follows:

> - Stop the Restore cron.

Reviewer: ```suggestion
- Stop the backup restore cron.
```

> - Start all the services on all the Frontend nodes.

Reviewer: ```suggestion
- Start the services on all the Automate and Chef Infra frontend nodes. `systemctl start chef-automate`
```

> - Update the DNS entry. Now DNS will point to the DR Load balancer.

Reviewer: ```suggestion
- Update the Automate FQDN DNS entry to resolve to the Disaster Recovery load balancer.
```

> - After the above steps, DR Cluster will be the primary cluster.

Reviewer: ```suggestion
- The Disaster Recovery cluster will be the primary cluster; it may take some time for DNS changes to fully propagate.
```

> - Need to set up the backup cron so that it will perform the backup.

Reviewer: ```suggestion
- Setup backup cron to start taking backups of the now active cluster.
```
